backend/templates/ma_sat_v1.txt

```diff
@@ -0,0 +1,21 @@
+{{ config(schema='@@Schema', materialized='view') }} 
+
+{%- set yaml_metadata -%}
+sat_v0: '@@SatName'
+hashkey: '@@Hashkey'
+hashdiff: '@@Hashdiff' 
+ma_attribute: 
+@@MaAttribute
+ledts_alias: 'valid_to'
```

Comment:
The default `ledts_alias` in datavault4dbt is `ledts`, is there a reason to deviate from that?

Reply:
Not sure about this one. Do you have any idea about this @rbesharat ?